diff --git a/remoting/android/java/src/org/chromium/chromoting/DummyClientExtension.java b/remoting/android/java/src/org/chromium/chromoting/DummyClientExtension.java
new file mode 100644
index 0000000000000000000000000000000000000000..cf942fa3e62e11715c25da7a530638a347894e3e
--- /dev/null
+++ b/remoting/android/java/src/org/chromium/chromoting/DummyClientExtension.java
@@ -0,0 +1,28 @@
+// Copyright 2014 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.chromium.chromoting;
+
+import android.app.Activity;
+
+/**
+ * A dummy implementation of ClientExtension.
+ */
+public class DummyClientExtension implements ClientExtension {
+
+    @Override
+    public String getCapability() {
+        return "";
+    }
+
+    @Override
+    public boolean onExtensionMessage(String type, String data) {
+        return false;
+    }
+
+    @Override
+    public ActivityLifecycleListener onActivityAcceptingListener(Activity activity) {
+        return null;
+    }
+}
