diff --git a/remoting/android/java/src/org/chromium/chromoting/TouchEventParameter.java b/remoting/android/java/src/org/chromium/chromoting/TouchEventParameter.java
new file mode 100644
index 0000000000000000000000000000000000000000..2d32f6575ded3fe5092f5bece0450c4b9030ef18
--- /dev/null
+++ b/remoting/android/java/src/org/chromium/chromoting/TouchEventParameter.java
@@ -0,0 +1,20 @@
+// Copyright 2016 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.chromium.chromoting;
+
+import android.view.MotionEvent;
+
+/**
+ * An {@link Event} parameter for a touch event.
+ */
+public final class TouchEventParameter {
+    public final MotionEvent event;
+    public boolean handled;
+
+    public TouchEventParameter(MotionEvent event) {
+        this.event = event;
+        this.handled = false;
+    }
+}
