diff --git a/chrome/browser/ui/signin_modal_dialog.cc b/chrome/browser/ui/signin_modal_dialog.cc
new file mode 100644
index 0000000000000000000000000000000000000000..21fb7e83f0f0e39d1b3e2490917ee11167f03476
--- /dev/null
+++ b/chrome/browser/ui/signin_modal_dialog.cc
@@ -0,0 +1,24 @@
+// Copyright 2021 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "chrome/browser/ui/signin_modal_dialog.h"
+
+#include <ostream>
+
+#include "base/check.h"
+
+SigninModalDialog::SigninModalDialog(base::OnceClosure on_close_callback)
+    : on_close_callback_(std::move(on_close_callback)) {
+  DCHECK(on_close_callback_);
+}
+
+SigninModalDialog::~SigninModalDialog() {
+  DCHECK(!on_close_callback_) << "NotifyModalDialogClosed() must have been "
+                                 "called before `this` is destroyed.";
+}
+
+void SigninModalDialog::NotifyModalDialogClosed() {
+  DCHECK(on_close_callback_);
+  std::move(on_close_callback_).Run();
+}
