diff --git a/extensions/browser/core_extensions_browser_api_provider.h b/extensions/browser/core_extensions_browser_api_provider.h
new file mode 100644
index 0000000000000000000000000000000000000000..cc2907d05620141eefe09059ed6d85e0edd0dae2
--- /dev/null
+++ b/extensions/browser/core_extensions_browser_api_provider.h
@@ -0,0 +1,26 @@
+// Copyright 2018 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EXTENSIONS_BROWSER_CORE_EXTENSIONS_BROWSER_API_PROVIDER_H_
+#define EXTENSIONS_BROWSER_CORE_EXTENSIONS_BROWSER_API_PROVIDER_H_
+
+#include "extensions/browser/extensions_browser_api_provider.h"
+
+namespace extensions {
+
+class CoreExtensionsBrowserAPIProvider : public ExtensionsBrowserAPIProvider {
+ public:
+  CoreExtensionsBrowserAPIProvider();
+  CoreExtensionsBrowserAPIProvider(const CoreExtensionsBrowserAPIProvider&) =
+      delete;
+  CoreExtensionsBrowserAPIProvider& operator=(
+      const CoreExtensionsBrowserAPIProvider&) = delete;
+  ~CoreExtensionsBrowserAPIProvider() override;
+
+  void RegisterExtensionFunctions(ExtensionFunctionRegistry* registry) override;
+};
+
+}  // namespace extensions
+
+#endif  // EXTENSIONS_BROWSER_CORE_EXTENSIONS_BROWSER_API_PROVIDER_H_
