diff --git a/extensions/browser/core_extensions_browser_api_provider.cc b/extensions/browser/core_extensions_browser_api_provider.cc
new file mode 100644
index 0000000000000000000000000000000000000000..23c75839dd67e52d7cd543128edb73eb89a329d1
--- /dev/null
+++ b/extensions/browser/core_extensions_browser_api_provider.cc
@@ -0,0 +1,19 @@
+// Copyright 2018 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "extensions/browser/core_extensions_browser_api_provider.h"
+
+#include "extensions/browser/api/generated_api_registration.h"
+
+namespace extensions {
+
+CoreExtensionsBrowserAPIProvider::CoreExtensionsBrowserAPIProvider() = default;
+CoreExtensionsBrowserAPIProvider::~CoreExtensionsBrowserAPIProvider() = default;
+
+void CoreExtensionsBrowserAPIProvider::RegisterExtensionFunctions(
+    ExtensionFunctionRegistry* registry) {
+  api::GeneratedFunctionRegistry::RegisterAll(registry);
+}
+
+}  // namespace extensions
