diff --git a/extensions/browser/service_worker_task_queue_factory.h b/extensions/browser/service_worker_task_queue_factory.h
new file mode 100644
index 0000000000000000000000000000000000000000..95264abb467def8989035d8db60abe1daa46eadd
--- /dev/null
+++ b/extensions/browser/service_worker_task_queue_factory.h
@@ -0,0 +1,40 @@
+// Copyright 2017 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EXTENSIONS_BROWSER_SERVICE_WORKER_TASK_QUEUE_FACTORY_H_
+#define EXTENSIONS_BROWSER_SERVICE_WORKER_TASK_QUEUE_FACTORY_H_
+
+#include "base/memory/singleton.h"
+#include "components/keyed_service/content/browser_context_keyed_service_factory.h"
+
+namespace extensions {
+
+class ServiceWorkerTaskQueue;
+
+class ServiceWorkerTaskQueueFactory : public BrowserContextKeyedServiceFactory {
+ public:
+  ServiceWorkerTaskQueueFactory(const ServiceWorkerTaskQueueFactory&) = delete;
+  ServiceWorkerTaskQueueFactory& operator=(
+      const ServiceWorkerTaskQueueFactory&) = delete;
+
+  static ServiceWorkerTaskQueue* GetForBrowserContext(
+      content::BrowserContext* context);
+  static ServiceWorkerTaskQueueFactory* GetInstance();
+
+ private:
+  friend struct base::DefaultSingletonTraits<ServiceWorkerTaskQueueFactory>;
+
+  ServiceWorkerTaskQueueFactory();
+  ~ServiceWorkerTaskQueueFactory() override;
+
+  // BrowserContextKeyedServiceFactory:
+  KeyedService* BuildServiceInstanceFor(
+      content::BrowserContext* context) const override;
+  content::BrowserContext* GetBrowserContextToUse(
+      content::BrowserContext* context) const override;
+};
+
+}  // namespace extensions
+
+#endif  // EXTENSIONS_BROWSER_SERVICE_WORKER_TASK_QUEUE_FACTORY_H_
