diff --git a/third_party/blink/renderer/core/css/css_position_fallback_rule.idl b/third_party/blink/renderer/core/css/css_position_fallback_rule.idl
new file mode 100644
index 0000000000000000000000000000000000000000..13a40cf1154c31ba1b1a9b3b16ad6a11f3f42dcb
--- /dev/null
+++ b/third_party/blink/renderer/core/css/css_position_fallback_rule.idl
@@ -0,0 +1,11 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(crbug.com/1309178): Define CSSOM API for @position-fallback at rule.
+
+[Exposed=Window, RuntimeEnabled=CSSAnchorPositioning]
+interface CSSPositionFallbackRule : CSSRule {
+  readonly attribute CSSOMString name;
+  readonly attribute CSSRuleList cssRules;
+};
