diff --git a/third_party/blink/renderer/core/css/font_face_source.cc b/third_party/blink/renderer/core/css/font_face_source.cc
new file mode 100644
index 0000000000000000000000000000000000000000..290a47ebbc416f197044080ef51ca880ebbfad9f
--- /dev/null
+++ b/third_party/blink/renderer/core/css/font_face_source.cc
@@ -0,0 +1,20 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "third_party/blink/renderer/core/css/font_face_source.h"
+
+#include "third_party/blink/renderer/core/css/font_face_set_document.h"
+#include "third_party/blink/renderer/core/css/font_face_set_worker.h"
+
+namespace blink {
+
+FontFaceSet* FontFaceSource::fonts(Document& document) {
+  return FontFaceSetDocument::From(document);
+}
+
+FontFaceSet* FontFaceSource::fonts(WorkerGlobalScope& worker) {
+  return FontFaceSetWorker::From(worker);
+}
+
+}  // namespace blink
