diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeBackupAgent.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeBackupAgent.java
new file mode 100644
index 0000000000000000000000000000000000000000..16f373f735f731cede0afbcb1ae75f9d15fdf6e7
--- /dev/null
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeBackupAgent.java
@@ -0,0 +1,18 @@
+// Copyright 2020 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.chromium.chrome.browser;
+
+import org.chromium.build.annotations.IdentifierNameString;
+import org.chromium.chrome.browser.base.SplitCompatBackupAgent;
+
+/** See {@link ChromeBackupAgentImpl}. */
+public class ChromeBackupAgent extends SplitCompatBackupAgent {
+    @IdentifierNameString
+    private static String sImplClassName = "org.chromium.chrome.browser.ChromeBackupAgentImpl";
+
+    public ChromeBackupAgent() {
+        super(sImplClassName);
+    }
+}
