diff --git a/content/browser/form_controls_browsertest_mac.mm b/content/browser/form_controls_browsertest_mac.mm
new file mode 100644
index 0000000000000000000000000000000000000000..c869ee4aceff4a8f513bcccb53cadb9cf96ebb51
--- /dev/null
+++ b/content/browser/form_controls_browsertest_mac.mm
@@ -0,0 +1,11 @@
+// Copyright 2021 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "content/browser/form_controls_browsertest_mac.h"
+
+bool MacOSVersionSupportsDarkMode() {
+  if (@available(macOS 10.14, *))
+    return true;
+  return false;
+}
