diff --git a/content/browser/form_controls_browsertest_mac.h b/content/browser/form_controls_browsertest_mac.h
new file mode 100644
index 0000000000000000000000000000000000000000..156c9764b2041f9fa87751e2085cfbca0656f39e
--- /dev/null
+++ b/content/browser/form_controls_browsertest_mac.h
@@ -0,0 +1,10 @@
+// Copyright 2021 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CONTENT_BROWSER_FORM_CONTROLS_BROWSERTEST_MAC_H_
+#define CONTENT_BROWSER_FORM_CONTROLS_BROWSERTEST_MAC_H_
+
+bool MacOSVersionSupportsDarkMode();
+
+#endif  // CONTENT_BROWSER_FORM_CONTROLS_BROWSERTEST_MAC_H_
