diff --git a/chrome/browser/focus_ring_browsertest_mac.h b/chrome/browser/focus_ring_browsertest_mac.h
new file mode 100644
index 0000000000000000000000000000000000000000..9bfaf9ae1d595cfc13948cf448e5c01f7e7e7fe8
--- /dev/null
+++ b/chrome/browser/focus_ring_browsertest_mac.h
@@ -0,0 +1,10 @@
+// Copyright 2020 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CHROME_BROWSER_FOCUS_RING_BROWSERTEST_MAC_H_
+#define CHROME_BROWSER_FOCUS_RING_BROWSERTEST_MAC_H_
+
+bool MacOSVersionSupportsDarkMode();
+
+#endif  // CHROME_BROWSER_FOCUS_RING_BROWSERTEST_MAC_H_
