diff --git a/third_party/blink/renderer/core/layout/collapsed_border_value.cc b/third_party/blink/renderer/core/layout/collapsed_border_value.cc
new file mode 100644
index 0000000000000000000000000000000000000000..a0a3c52125074c5f30da376dd52b73ffa0f46d74
--- /dev/null
+++ b/third_party/blink/renderer/core/layout/collapsed_border_value.cc
@@ -0,0 +1,34 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "third_party/blink/renderer/core/layout/collapsed_border_value.h"
+
+#include "third_party/blink/renderer/core/style/computed_style.h"
+
+namespace blink {
+
+CollapsedBorderValue::CollapsedBorderValue(const BorderValue& border,
+                                           const Color& color,
+                                           EBorderPrecedence precedence)
+    : CollapsedBorderValue(border.Style(), border.Width(), color, precedence) {}
+
+CollapsedBorderValue::CollapsedBorderValue(EBorderStyle style,
+                                           const LayoutUnit width,
+                                           const Color& color,
+                                           EBorderPrecedence precedence)
+    : color_(color),
+      style_(static_cast<unsigned>(style)),
+      precedence_(precedence) {
+  if (!ComputedStyle::BorderStyleIsVisible(style)) {
+    width_ = 0;
+  } else {
+    if (width > 0 && width <= 1)
+      width_ = 1;
+    else
+      width_ = width.ToUnsigned();
+  }
+  DCHECK(precedence != kBorderPrecedenceOff);
+}
+
+}  // namespace blink
