diff --git a/third_party/blink/renderer/core/layout/layout_inside_list_marker.cc b/third_party/blink/renderer/core/layout/layout_inside_list_marker.cc
new file mode 100644
index 0000000000000000000000000000000000000000..d18bce73b675dfb4c6db52d26a3ed95929d84ae1
--- /dev/null
+++ b/third_party/blink/renderer/core/layout/layout_inside_list_marker.cc
@@ -0,0 +1,14 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "third_party/blink/renderer/core/layout/layout_inside_list_marker.h"
+
+namespace blink {
+
+LayoutInsideListMarker::LayoutInsideListMarker(Element* element)
+    : LayoutInline(element) {}
+
+LayoutInsideListMarker::~LayoutInsideListMarker() = default;
+
+}  // namespace blink
