diff --git a/third_party/blink/renderer/core/dom/idle_request_options.idl b/third_party/blink/renderer/core/dom/idle_request_options.idl
new file mode 100644
index 0000000000000000000000000000000000000000..9764baabab53522359a407fee60b4ed3b9917b6d
--- /dev/null
+++ b/third_party/blink/renderer/core/dom/idle_request_options.idl
@@ -0,0 +1,9 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// https://w3c.github.io/requestidlecallback/#window_extensions
+
+dictionary IdleRequestOptions {
+    unsigned long timeout = 0;
+};
