diff --git a/extensions/common/draggable_region.h b/extensions/common/draggable_region.h
new file mode 100644
index 0000000000000000000000000000000000000000..aafce0b0fd9fc43fe8497c0360d167e4b87404bf
--- /dev/null
+++ b/extensions/common/draggable_region.h
@@ -0,0 +1,21 @@
+// Copyright 2012 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EXTENSIONS_COMMON_DRAGGABLE_REGION_H_
+#define EXTENSIONS_COMMON_DRAGGABLE_REGION_H_
+
+#include "ui/gfx/geometry/rect.h"
+
+namespace extensions {
+
+struct DraggableRegion {
+  bool draggable;
+  gfx::Rect bounds;
+
+  DraggableRegion();
+};
+
+}  // namespace extensions
+
+#endif  // EXTENSIONS_COMMON_DRAGGABLE_REGION_H_
