diff --git a/extensions/common/extension_message_generator.cc b/extensions/common/extension_message_generator.cc
new file mode 100644
index 0000000000000000000000000000000000000000..52183009d6dabb51cdf25f19a5881a7feb86b72d
--- /dev/null
+++ b/extensions/common/extension_message_generator.cc
@@ -0,0 +1,30 @@
+// Copyright 2014 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Get basic type definitions.
+#define IPC_MESSAGE_IMPL
+#include "extensions/common/extension_message_generator.h"
+
+// Generate constructors.
+#include "ipc/struct_constructor_macros.h"
+#include "extensions/common/extension_message_generator.h"
+
+// Generate param traits write methods.
+#include "ipc/param_traits_write_macros.h"
+namespace IPC {
+#include "extensions/common/extension_message_generator.h"
+}  // namespace IPC
+
+// Generate param traits read methods.
+#include "ipc/param_traits_read_macros.h"
+namespace IPC {
+#include "extensions/common/extension_message_generator.h"
+}  // namespace IPC
+
+// Generate param traits log methods.
+#include "ipc/param_traits_log_macros.h"
+namespace IPC {
+#include "extensions/common/extension_message_generator.h"
+}  // namespace IPC
+
