diff --git a/chrome/browser/ui/extensions/extension_install_ui_factory.h b/chrome/browser/ui/extensions/extension_install_ui_factory.h
new file mode 100644
index 0000000000000000000000000000000000000000..f8eb7faecc7d73c69f01e45de9c862302ed11087
--- /dev/null
+++ b/chrome/browser/ui/extensions/extension_install_ui_factory.h
@@ -0,0 +1,22 @@
+// Copyright 2014 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CHROME_BROWSER_UI_EXTENSIONS_EXTENSION_INSTALL_UI_FACTORY_H_
+#define CHROME_BROWSER_UI_EXTENSIONS_EXTENSION_INSTALL_UI_FACTORY_H_
+
+#include <memory>
+
+namespace content {
+class BrowserContext;
+}
+
+namespace extensions {
+class ExtensionInstallUI;
+
+std::unique_ptr<extensions::ExtensionInstallUI> CreateExtensionInstallUI(
+    content::BrowserContext* context);
+
+}  // namespace extensions
+
+#endif  // CHROME_BROWSER_UI_EXTENSIONS_EXTENSION_INSTALL_UI_FACTORY_H_
