diff --git a/base/android/base_features.h b/base/android/base_features.h
new file mode 100644
index 0000000000000000000000000000000000000000..653e80b59379b14dd3d4f3fafe522dcb0175713d
--- /dev/null
+++ b/base/android/base_features.h
@@ -0,0 +1,22 @@
+// Copyright 2022 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef BASE_ANDROID_BASE_FEATURES_H_
+#define BASE_ANDROID_BASE_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace base::android::features {
+
+// All features in alphabetical order. The features should be documented
+// alongside the definition of their values in the .cc file.
+
+// Alphabetical:
+extern const base::Feature kBrowserProcessMemoryPurge;
+extern const base::Feature kCrashBrowserOnChildMismatchIfBrowserChanged;
+extern const base::Feature kCrashBrowserOnAnyChildMismatch;
+
+}  // namespace base::android::features
+
+#endif  // BASE_ANDROID_BASE_FEATURES_H_
