diff --git a/base/android/base_features.cc b/base/android/base_features.cc
new file mode 100644
index 0000000000000000000000000000000000000000..ef55e1b147a9c06b1d07d6b05c276b622f339e12
--- /dev/null
+++ b/base/android/base_features.cc
@@ -0,0 +1,31 @@
+// Copyright 2022 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "base/android/base_features.h"
+#include "base/feature_list.h"
+
+namespace base::android::features {
+
+// Alphabetical:
+
+// When the browser process has been in the background for several minutes at a
+// time, trigger an artificial critical memory pressure notification. This is
+// intended to reduce memory footprint.
+const base::Feature kBrowserProcessMemoryPurge{"BrowserProcessMemoryPurge",
+                                               FEATURE_DISABLED_BY_DEFAULT};
+
+// Crash the browser process if a child process is created which does not match
+// the browser process and the browser package appears to have changed since the
+// browser process was launched, so that the browser process will be started
+// fresh when next used, hopefully resolving the issue.
+const base::Feature kCrashBrowserOnChildMismatchIfBrowserChanged{
+    "CrashBrowserOnChildMismatchIfBrowserChanged", FEATURE_DISABLED_BY_DEFAULT};
+
+// Crash the browser process if a child process is created which does not match
+// the browser process regardless of whether the browser package appears to have
+// changed.
+const base::Feature kCrashBrowserOnAnyChildMismatch{
+    "CrashBrowserOnAnyChildMismatch", FEATURE_DISABLED_BY_DEFAULT};
+
+}  // namespace base::android::features
