diff --git a/base/android/reached_code_profiler_stub.cc b/base/android/reached_code_profiler_stub.cc
new file mode 100644
index 0000000000000000000000000000000000000000..b0f27a51e85d8c1abe7b40515b9d6c751f83e1b0
--- /dev/null
+++ b/base/android/reached_code_profiler_stub.cc
@@ -0,0 +1,22 @@
+// Copyright 2019 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "base/android/reached_code_profiler.h"
+
+namespace base {
+namespace android {
+
+void InitReachedCodeProfilerAtStartup(LibraryProcessType library_process_type) {
+}
+
+bool IsReachedCodeProfilerEnabled() {
+  return false;
+}
+
+bool IsReachedCodeProfilerSupported() {
+  return false;
+}
+
+}  // namespace android
+}  // namespace base
