diff --git a/chrome/browser/extensions/chrome_extension_browser_constants.cc b/chrome/browser/extensions/chrome_extension_browser_constants.cc
new file mode 100644
index 0000000000000000000000000000000000000000..5eefc5c2b4f0bb489c6191e270b6844182a27abf
--- /dev/null
+++ b/chrome/browser/extensions/chrome_extension_browser_constants.cc
@@ -0,0 +1,25 @@
+// Copyright 2018 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "chrome/browser/extensions/chrome_extension_browser_constants.h"
+
+namespace chrome_extension_constants {
+
+// The link to the help article for runtime host permissions.
+const char kRuntimeHostPermissionsHelpURL[] =
+    "https://support.google.com/chrome?p=enable_extensions";
+
+// The link to the site permissions settings page.
+const char kExtensionsSitePermissionsURL[] =
+    "chrome://extensions/sitePermissions";
+
+// The link to the help article for click to script, which contains info on
+// enabling extensions to request access to the current site through the
+// toolbar.
+// TODO(crbug.com/1353587): This link is likely temporary, and may need to be
+// replaced later.
+const char kShowAccessRequestsInToolbarHelpURL[] =
+    "https://support.google.com/chrome_webstore/answer/2664769";
+
+}  // namespace chrome_extension_constants
